push/model/src/test/resources/message-format.json

```diff
@@ -5,6 +5,7 @@
     "alert":"HELLO!",
     "sound":"default",
     "badge":2,
+    "page":"/MainPage.xaml",
```

Comment:
@lfryc @edewit do we have a JIRA ticket to document this message format change ? e.g. on RESTful endpoint ?